packages/core/test/core.test.ts

```diff
@@ -0,0 +1,50 @@
+/* tslint:disable */
+const mock = require('mam.tools.js/test/mamMock');
+const { log } = require('mam.tools.js/lib/logger');
+
+import { composeAPI } from '../src';
+import { publicKeyPem, privateKeyPem } from './keypair';
+import { generateSeed } from './utilities';
+
+// disable mam.tools.js console logging
+log.silent = true;
+
+jest.mock(
+  'mam.client.js',
+  () => {
+    return mock;
+  },
+  { virtual: true }
+);
+
+describe('@tangleid/core', () => {
+  it('register identifier and generate a verifiable credential', async () => {
+    const tid = composeAPI({
+      providers: {
+        '0x1': 'http://node.deviceproof.org:14265',
+        '0x2': 'https://nodes.devnet.thetangle.org:443',
```

Comment:
Are we still going to use testnet node?

Reply:
Remove it. We do not recommend to use the testnet node for now.